tests/mcmc/test_sampling.py

```diff
@@ -270,7 +270,7 @@ def test_linear_regression_sgld(self):
 
         data_batch = X_data[:100, :]
         init_position = 1.0
-        _ = sgld(rng_key, init_position, data_batch, 1e-3)
+        _ = sgld.step(rng_key, init_position, data_batch, 1e-3)
```

Comment:
Also what about adding `init_position = sgld.init(init_position)` to keep the coverage up?